protocol/avdl/chat1/common.avdl

```diff
@@ -276,6 +281,10 @@
     Expunge expunge;
     union { null, RetentionPolicy } convRetention;
     union { null, RetentionPolicy } teamRetention;
+
+    // The last message id that was exploded.
```

Comment:
Is my understanding of this right:

> This id isn't necessarily increasing. An older id might explode after a newer one. Clients should only ever check this value for equality, rather than > or <.

If so, maybe add that above to the comment here?

Reply:
Oh actually, the comment above says "last message id", but the field we have here is a `keybase1.Time`. Should the comment say "time"? Will times be comparable between the client and the server?

Reply:
yeah comments are out of date, i'll fix. we just compare the current state to what the server sent and if they differ we attempt a purge so we're not comparing client time to server time